docs/src/content/docs/guides/create-interactive-grant.mdx

```diff
@@ -31,80 +32,143 @@ Although grants for creating incoming payments and quotes are non-interactive by
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='1'
+/>
 
-The URI will be provided by the authorization server in its response.
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='2'
+/>
 
-```ts
-interact: {
-    start: ["redirect"],
-},
-```
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='3'
+/>
+
+#### Request an interactive grant
+
+Indicate your client is capable of directing the end-user to a URI to start an interaction. We’ll use an outgoing payment grant as an example.
 
-### 2. Specify a redirect finish mode
+1. Add the `interact` object to your grant request.
+2. Specify `redirect` as the `start` mode. Open Payments only supports `redirect` at this time. The redirect URI will be provided by the authorization server in its response.
 
-Indicate your client can receive a signal from the authorization server when the interaction has finished.
+Now, indicate your client can receive a signal from the authorization server when the interaction has finished.
 
-1. Add the `finish` object within the existing `interact` object.
-2. Specify `redirect` as the `method`. Open Payments only supports `redirect`.
+1. Add the `finish` object within the `interact` object.
+2. Specify `redirect` as the `method`. Open Payments only supports `redirect` at this time.
 3. Specify the `uri` that the authorization server will send the end-user back to when the interaction has finished.
 4. Create a `nonce` for your client to use to verify the authorization server is the same entity throughout the entire process. The nonce can be any random, hard-to-guess string.
 
-<CodeBlock title="Example">
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='4'
+/>
 
-```ts
-interact: {
-    start: ["redirect"],
-    finish: {
-        method: "redirect",
-        uri: "https://example.wallet.com/return/876FGRD8VC",
-        nonce: 4edb2194-dbdf-46bb-9397-d5fd57b7c8a7,
-    },
-},
-```
-
-</CodeBlock>
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='5'
+/>
 
-### 3. Interact with the end user
+#### Start interaction with the end user
 
-#### Start interaction
+Once your client receives the authorization server’s response, it must send the end-user to the `interact.redirect` URI contained in the response. This starts the interaction flow.
 
-Once your client receives the [authorization server’s response](/apis/auth-server/operations/post-request/#responses), it must send the end-user to the `interact redirect` URI contained in the response. This initiates the interaction flow.
+<CodeBlock title="Example response">
```

Comment:
In the preview, there's white space between this code block's title and body. I don't know why it's happening, so I'll need to investigate more.